actors/migration/nv9/top.go

```diff
@@ -241,12 +258,19 @@ func (job *migrationJob) run(ctx context.Context, store cbor.IpldStore, priorEpo
 		balance:    job.Actor.Balance,
 		head:       job.Actor.Head,
 		priorEpoch: priorEpoch,
+		cache:      job.cache,
```

Comment:
Since the cache is available here, the cached migrator isn't abstracting anything. If retaining this pattern of per-actor-instance caches (see comments elsewhere), just ditch the `cachedMigrator` and inline the cache lookup here. Use the computing map pattern to encapsulate the write at L269 too.

But I think it would be better again to make a singleton cache a member of and hence encapsulated by the `cachedMigrator`: do both the reading and the writing to the cache in the same method, in `cachedMigrator`.